### DISCUSSION TITLE
[Question] Assuming I use unlimited value and run into issues, which issues would those be ?

### DISCUSSION THREAD

**Discussion by flyingturtl3** [xCuri0/ReBarUEFI#85] [Q&A] [answered]
Quoting "In most cases you should be able to use 32 (unlimited) without issues but you might need to use a smaller BAR size if 32 doesn't work", what would the symptoms be if it doesn't work or have issues ?

**Comment by xCuri0** [comment, marked as answer]
@flyingturtl3 no boot because you have something like a rtx 3090 which uses 32gb on ivy bridge where the bios won't support that.

CMOS reset (take out battery in some cases) will fix it.
